### PR TITLE
updated start Date and no Of days

### PR DESCRIPTION
**Pull request by sendilkumarn**
#60

### CODE REVIEW

File: angular-material-calendar.js [hunk 1]
```diff
@@ -28,11 +28,33 @@ angular.module("materialCalendar").service("materialCalendar.Calendar", [functio
...
+            }
+            return this.noOfDays;
+        };
+
         this.options = angular.isObject(options) ? options : {};
         this.year = now.getFullYear();
         this.month = now.getMonth();
         this.weeks = [];
         this.weekStartsOn = this.setWeekStartsOn(this.options.weekStartsOn);
+        this.startDate = this.setStartDate(this.options.startDate);
```
Comment (gavinbarron) [inline angular-material-calendar.js]: Perhaps this is better called startDayOfMonth? startDate suggest a value like '2016-01-31'

Comment (sendilkumarn) [inline angular-material-calendar.js]: Nited. Will change it

File: src/angular-material-calendar.js [hunk 2]
```diff
@@ -68,11 +90,30 @@ angular.module("materialCalendar").service("materialCalendar.Calendar", [functio
...
-                monthLength++;
+            if ( date.getDate() === 1) {
+                while ( date.getDay() !== this.weekStartsOn) {
+                    date.setDate(date.getDate() - 1);
+                    monthLength++;
+                }
+            }
+
+            if ( this.noOfDays !== 0) {
+              while (this.noOfDays % 7 !== 0) {
```
Comment (gavinbarron) [inline src/angular-material-calendar.js]: CI is complaining about too many spaces here

File: src/angular-material-calendar.js [hunk 3]
```diff
@@ -68,11 +90,30 @@ angular.module("materialCalendar").service("materialCalendar.Calendar", [functio
...
+                  this.noOfDays++;
+              }
+              monthLength = this.noOfDays;
```
Comment (gavinbarron) [inline src/angular-material-calendar.js]: CI is complaining about too many spaces here

**Comment by sendilkumarn** [comment]
@gavinbarron will you be adding this?

**Pull request merged**